build/tools/travis-ci.sh

```diff
@@ -73,6 +73,11 @@ case $wxTOOLSET in
         make -C tests $wxJOBS $wxMAKEFILE_FLAGS
         echo -en 'travis_fold:end:script.tests\\r'
 
+        if [ "$wxSKIP_TESTING" = 1 ]; then
+            echo 'Skipping running tests'
+            exit 0
```

Comment:
If possible, I'd like to try at least building the minimal sample, as otherwise we wouldn't detect any breakage resulting in linking errors.

Reply:
I'm only skipping **running** the test, test and test_gui are still built, would this not bet catching the linking errors as well ?

Reply:
Oops, sorry, I've just misread this in my haste. This is indeed fine, sorry again for the false alarm.